package.json

```diff
@@ -7,7 +7,9 @@
     "start": "cross-env NODE_ENV=development webpack-serve",
     "build": "cross-env NODE_ENV=deployment webpack",
     "eslint": "eslint src --fix",
-    "test": "node $NODE_DEBUG_OPTION node_modules/.bin/ava"
+    "test1": "node $NODE_DEBUG_OPTION node_modules/.bin/ava",
```

Comment:
please remove any ava configuration, files, and dependencies from the project.

Reply:
done